config/manager/network_policy.yaml

```diff
@@ -2,11 +2,28 @@
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: prevent-all-egress
+  name: prevent-all-egress-allow-api-server
   namespace: system
 spec:
   podSelector:
     matchLabels:
-      control-plane: controller-manager
+      control-plane: controller-man
```

Comment:
Is this intentional? It looks like a mistake.

Reply:
This is a mistake , I by mistake deleted the label value, it should be controller-manager